iroha/config.json

```diff
@@ -1,11 +1,25 @@
 {
-  "TORII_URL": "127.0.0.1:1337",
-  "BLOCK_TIME_MS": 1000,
-  "COMMIT_TIME_MS": 1000,
-  "TX_RECEIPT_TIME_MS": 100,
-  "KURA_INIT_MODE": "strict",
-  "KURA_BLOCK_STORE_PATH": "./blocks",
-  "MAX_FAULTY_PEERS": 1,
+  "TORII_CONFIGURATION": {
+    "TORII_URL": "127.0.0.1:1337"
+  },
+  "SUMERAGI_CONFIGURATION": {
+    "BLOCK_TIME_MS": 1000,
+    "COMMIT_TIME_MS": 1000,
+    "TX_RECEIPT_TIME_MS": 100,
+    "MAX_FAULTY_PEERS": 1,
+    "KEY_PAIR": {
+      "public_key": {"inner": [101, 170, 80, 164, 103, 38, 73, 61, 223, 133, 83, 139, 247, 77, 176, 84, 117, 15, 22, 28, 155, 125, 80, 226, 40, 26, 61, 248, 40, 159, 58, 53]},
+      "private_key": {"inner": [113, 107, 241, 108, 182, 178, 31, 12, 5, 183, 243, 184, 83, 0, 238, 122, 77, 86, 20, 245, 144, 31, 128, 92, 166, 251, 245, 106, 167, 188, 20, 8,     101, 170, 80, 164, 103, 38, 73, 61, 223, 133, 83, 139, 247, 77, 176, 84, 117, 15, 22, 28, 155, 125, 80, 226, 40, 26, 61, 248, 40, 159, 58, 53]}
+    },
+    "PEER_ID": {
+      "address": "127.0.0.1:1337",
+      "public_key": {"inner": [101, 170, 80, 164, 103, 38, 73, 61, 223, 133, 83, 139, 247, 77, 176, 84, 117, 15, 22, 28, 155, 125,     80, 226, 40, 26, 61, 248, 40, 159, 58, 53]}
+    }
```

Comment:
We have duplication of `public_key` field. It may lead to errors in configuration.